README.rst

```diff
@@ -138,8 +138,7 @@ the topic, and subscribe to that, passing a callback function.
         project_id=os.getenv('GOOGLE_CLOUD_PROJECT'),
         sub='MY_SUBSCRIPTION_NAME',  # Set this to something appropriate.
     )
-    subscriber.create_subscription(
-        name=subscription_name, topic=topic_name)
+    subscriber.create_subscription(request={"name": subscription_name, "topic": topic_name})
```

Comment:
Same thing here - `create_subscription()` supports subscription name and topic name as keyword arguments ([source](https://github.com/googleapis/python-pubsub/blob/7228f6c9a4c050bf22bb4bc3582b89b04eaa8702/google/pubsub_v1/services/subscriber/client.py#L298-L309)).